Homework30/ServerSide/src/main/java/org/example/ServerHW.java

```diff
@@ -0,0 +1,132 @@
+package org.example;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class ServerHW {
+    private static final int PORT = 3345;
+    private static final String SERVER_NAME = "Server";
+    private static final String CLIENT_NAME = "client-";
+    private static final Logger LOGGER = Logger.getLogger(ServerHW.class.getName());
+
+    private List<ClientHandler> clients;
+
+    public static void main(String[] args) {
+        ServerHW server = new ServerHW();
+        server.start();
+    }
+
+    public ServerHW() {
+        clients = new ArrayList<>();
+    }
+
+    public void start() {
+        try {
+            ServerSocket serverSocket = new ServerSocket(PORT);
+            LOGGER.info("Server started on port " + PORT);
+
+            while (true) {
+                Socket clientSocket = serverSocket.accept();
+                LOGGER.info("New client connected: " + clientSocket);

```

Comment:
Do not print the whole socket information! At this point, you're supposed to get the client's name and pass it to the logger.